CHANGELOG.md

```diff
@@ -10,11 +10,6 @@
 ### Breaking changes
 * None.
 
------------
```

Comment:
Link above doesn't work: 3282 -> 3182
Adding which method is failing would be good. (Group::get_used_space()).
For example:

* A segmentation fault would occur when calling Group:get_used_space() for a realm file with no commits. This method would usually only be called from sync/ROS to calculate and report state size. ([#3182](https://github.com/realm/realm-core/issues/3182), since v5.12.0)